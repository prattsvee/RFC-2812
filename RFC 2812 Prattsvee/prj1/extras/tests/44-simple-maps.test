%{ a: 22, :b => 33 }

%{ [22] => 33, {:x} => :x } %{}

%{ 22 => # ignore
  33
}