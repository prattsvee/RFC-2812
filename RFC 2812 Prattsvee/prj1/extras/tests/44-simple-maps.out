[
  {
    "%k": "map",
    "%v": [
      [
        {
          "%k": "atom",
          "%v": ":a"
        },
        {
          "%k": "int",
          "%v": 22
        }
      ],
      [
        {
          "%k": "atom",
          "%v": ":b"
        },
        {
          "%k": "int",
          "%v": 33
        }
      ]
    ]
  },
  {
    "%k": "map",
    "%v": [
      [
        {
          "%k": "list",
          "%v": [
            {
              "%k": "int",
              "%v": 22
            }
          ]
        },
        {
          "%k": "int",
          "%v": 33
        }
      ],
      [
        {
          "%k": "tuple",
          "%v": [
            {
              "%k": "atom",
              "%v": ":x"
            }
          ]
        },
        {
          "%k": "atom",
          "%v": ":x"
        }
      ]
    ]
  },
  {
    "%k": "map",
    "%v": []
  },
  {
    "%k": "map",
    "%v": [
      [
        {
          "%k": "int",
          "%v": 22
        },
        {
          "%k": "int",
          "%v": 33
        }
      ]
    ]
  }
]
